acceptance-tests/lib/testState.ts

```diff
@@ -23,14 +28,26 @@ export class TestState {
     return this.config?.personalAccessKey;
   }
 
-  getTestConfigFileName() {
+  getTestConfigFileNameRelativeToOutputDir() {
     return this.testConfigFileName;
   }
 
+  getTestConfigFileName() {
+    return this.getPathWithinTestDirectory(this.testConfigFileName);
+  }
+
+  getPathWithinTestDirectory(filepath: string) {
+    return path.join(this.testOutputDir, filepath);
+  }
+
+  existsInProjectFolder(filepath: string) {
```

Comment:
What's the word "project" referring to here? A generic project or a literal hubspot developer project?

Reply:
I went with `existsInTestOutputDirectory`